### ISSUE TITLE
The shape of logits

### ISSUE THREAD

**Issue by DIVE128** [jiawei-ren/BalancedMSE#9]
The shape of logits in **bmc loss** is [batch, batch] which means both pred and target have the size [batch, 1].  **For dense prediction, do we have to flat the 2d-image to a 1d vector with an operation of unsqueeze(-1)?**

**Comment by jiawei-ren** [comment]
Yes, that's right, you may refer to the code in [NYUD2-DIR](https://github.com/jiawei-ren/BalancedMSE/blob/main/nyud2-dir/train.py#L233). Also, be aware that as mentioned in the paper Sect. 4.2.3, GAI/BNI may have a better performance than BMC, and manual noise sigma tuning is recommended.
